### PR TITLE
fix(inputs.diskio): Add missing udev properties

### LINKED ISSUE #3663: Expected udev properties cannot be used in diskio input
### Relevant telegraf.conf:

[[inputs.diskio]]
  device_tags = ["DEVTYPE", "ID_FS_TYPE"]

### System info:

Telegraf v1.5.0 (git: release-1.5 a1668bbf)
CentOS Linux release 7.4.1708 (Core)

### Steps to reproduce:

1. Add "device_tags = ["DEVTYPE", "ID_FS_TYPE"]" to the [[inputs.diskio]] section
2. udevadm info -q property -n /dev/sda1 | egrep 'DEVTYPE|ID_FS_TYPE'

```
DEVTYPE=partition
ID_FS_TYPE=linux_raid_member
```

3. telegraf --test | grep sda1 | sed 's|,|,\n|g;s| |\n|g'

```
2018/01/12 12:15:24 I! Using config file: /etc/telegraf/telegraf.conf
>
diskio,
name=sda1,
ID_FS_TYPE=linux_raid_member,
host=example.com
reads=1810738130i,
read_bytes=44825536246784i,
write_time=502432468i,
io_time=196321694i,
iops_in_progress=0i,
writes=158223504i,
write_bytes=1859747491328i,
read_time=1446620992i,
weighted_io_time=1948537468i
1515759325000000000
```

### Expected behavior:

DEVTYPE and ID_FS_TYPE tags in the telegraf output

### Actual behavior:

ID_FS_TYPE only

### Additional info:

telegraf ignoring all device_tags in the [[inputs.diskio]] section which does not contain underscore: DEVNAME, DEVTYPE, SUBSYSTEM, etc.

### PR DESCRIPTION
**Pull request by srebhan**
## Summary

Add the missing `DEVPATH`, `DEVNAME` and `DEVTYPE` properties. In the course also fix the sample.conf to represent the default values.

## Checklist

- [x] No AI generated code was used in this PR

## Related issues

resolves #3663

### CODE REVIEW

**Review by powersj — APPROVED** [review]
+1, one question about the readme.

File: plugins/inputs/diskio/README.md [hunk 1]
```diff
@@ -16,24 +16,20 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
 ```toml @sample.conf
 # Read metrics about disk IO by device
 [[inputs.diskio]]
-  ## By default, telegraf will gather stats for all devices including
-  ## disk partitions.
-  ## Setting devices will restrict the stats to the specified devices.
-  ## NOTE: Globbing expressions (e.g. asterix) are not supported for
-  ##       disk synonyms like '/dev/disk/by-id'.
```
[the remark this answers is not in the document]
Comment (srebhan) [reply plugins/inputs/diskio/README.md:23]: Hmm line 20 says:

> ## Wildcards are supported except for disk synonyms like '/dev/disk/by-id'.

Isn't this basically the same to what was there before?

[the remark this answers is not in the document]
Comment (powersj) [reply plugins/inputs/diskio/README.md:23]: ugh! yes how the heck did miss this... :facepalm:

**Review by DStrand1 — APPROVED** [review]

**Comment by Daryes** [comment]
Sorry to chime in a little late, but unless I am mistaken, the default value for `skip_serial_number` in the code is : false.
The updated documentation currently states the default is : true

**Comment by powersj** [comment]
@Daryes,

While booleans default to false, when we initialize the plugin it defaults to true:

https://github.com/influxdata/telegraf/blob/master/plugins/inputs/diskio/diskio.go#L192

Let me know if you are seeing otherwise.

**Comment by Daryes** [comment]
I didn't see the init bloc while searching in the code.
Thanks

**Pull request merged**